react/Reviews.tsx

```diff
@@ -533,6 +558,170 @@ const Reviews: FunctionComponent<InjectedIntlProps & Props> = props => {
           </Fragment>
         )}
       </div>
+      {state.settings.showGraph ? (
+        <div className={`${handles.graphContent} mv5`}>
```

Comment:
Create a separate component for the graph

Reply:
Done :)
Link active on: https://longo--biasicom.myvtex.com/aspirator-cu-sac-samsung-vc07m25e0wr-2-5-l-700-w-tub-telescopic-accesorii-2-in-1-rosu/p